full-node/db/sov-db/src/native_db.rs

```diff
@@ -31,29 +36,72 @@ impl NativeDB {
             &gen_rocksdb_options(&Default::default(), false),
         )?;
 
+        let next_version = Self::last_version_written(&inner)?.unwrap_or_default() + 1;
         Ok(Self {
             db: Arc::new(inner),
+            next_version: Arc::new(Mutex::new(next_version)),
```

Comment:
This is the same version as in `StateDB` right? Shouldn't we accept ` If yes should we share it between `StateDB ` and `NativeDB ` and inceremnt only in one place?